#163405222 create a landing page

vkingmaker opened the pull request:
#### What does this PR do?
- Add a landing page to the project to from where users can login or signup to use the application
#### Description of Task to be completed?
`/GET /index.html`
#### How should this be manually tested?
- After cloning the repository
- `cd` into the project.
- run a local server to host the app locally. 
- using your favorite browser visit the link above.
#### What are the relevant pivotal tracker stories?
#163405222
#### Screenshots
![landingpage](https://user-images.githubusercontent.com/18199741/51605093-50604380-1f0e-11e9-8299-f0d609e067b5.PNG)

vkingmaker committed:
[Feature #163405222] create a landing page
This PR adds a landing page to the application with a link to login or singup
[Delivers #16340522]

vkingmaker committed:
feature(landingPage): create a landingPage

This PR adds a landing page to the application with a link to login or singup

[Delivers #16340522]

megame24 commented on UI/assets/css/main.css:
```diff
@@ -0,0 +1,151 @@
+    width: 60%;
+    height: 50%;
+    border-radius: 50%;
+} */
```
Get rid of commented blocks of code.

megame24 commented on UI/assets/css/main.css:
```diff
@@ -0,0 +1,151 @@
+    padding: 1.1rem;
+    text-align: justify;
+    line-height: 1.2rem;
+} */
```
Here too

megame24 commented:
Your commit message name still is not following the recommended convention, please refer to the resource shared earlier.

vkingmaker committed:
[Feature #163405222] create a landing page
This PR adds a landing page to the application with a link to login or singup
[Delivers #16340522]

vkingmaker committed:
feature (landingPage): create a landingPage

This PR creates a landingPage with login and signup links

[Delivers #163405222]

vkingmaker committed:
feature(landingPage): create a landingPage

create a landingPage for the application with signup and login links (squashed commits)

[Delivers #163405222]

megame24 commented:
Instead of creating new commits in an attempt to rename a commit, make use of `git commit amend`. Also, learn how to reach out whenever you are facing a blocker.

megame24 reviewed (approved):
Looks good, you can merge
